site/docs/v1/tech/apis/_application-request-body.adoc

```diff
@@ -436,6 +436,9 @@ The value sent in the AuthN response to the SAML v2 Service Provider in the Name
 [field]#application.samlv2Configuration.keyId# [type]#[UUID]# [optional]#Optional# [default]#Defaults to a new key# [since]#Available since 1.6.0#::
 The unique Id of the Key used to sign the SAML response. If you do not specify this property, FusionAuth will create a new key and associate it with this Application.
 
+[field]#application.samlv2Configuration.loginHintParameter# [type]#[String]# [optional]#Optional# [since]#Available since 1.47.0#::
```

Comment:
Is there a default value?

Reply:
No, default is no login hint since every SP seems to do this differently.